examples/connext_dds/xml_application_env/c++11/application.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+
```

Comment:
Could you add the copyright heading to this file as well?

Reply:
Certainly.  I'll throw in a decent file header comment block as well :)

Reply:
Certainly. I will add a nice file block comment as well.  :)